README.md

```diff
@@ -213,7 +213,7 @@ Vimeo | <https://vimeo.com> | ✓ | | |
 ## Known issues
 > net/http: request canceled (Client.Timeout exceeded while reading body)
 
-You may have had this problem. It's not a bug, it's the network issue. Just download it again and everything will get back.
+This is a common issue. It is a network issue that can be solved by redownloading Annie.
```

Comment:
`redownloading Annie`?

Reply:
I don't fully understand what you meant by
> Just download it again and everything will get back

I understood it as "just download annie again and the error will be gone" ?

Reply:
Sorry for my poor English, I will explain it later. By the way, you are native English speaker?

Reply:
It's okay! and yes, English is my first language

Reply:
hey @xjohn600, I'm back

> Just download it again and everything will get back

I mean just execute the previous command again(download it again), everything will turn normal.

Reply:
Ohh okay. I revised it to

> This is a common issue. It is a network issue that can be solved by entering the download command again

Would that be ok?

Reply:
cool!